packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -38,15 +38,15 @@ const Web3 = require("web3");
 
 const { Logger } = require("@uma/financial-templates-lib");
 let logger;
-let protocolRunnerUrl;
+let spokeUrl;
 let customNodeUrl;
 let hubConfig = { configRetrieval: "localStorage", saveQueriedBlock: "localStorage", spokeRunner: "localStorage" };
 
 hub.post("/", async (req, res) => {
   try {
     logger.debug({
-      at: "CloudRunHub",
```

Comment:
rename all refrences to `CloudRun` to `Serverless` to make it more generic and cleaner.